apps/arweave/src/arweave_app.erl

```diff
@@ -0,0 +1,20 @@
+-module(arweave_app).
+-behaviour(application).
+
+-export([start/2, stop/1]).
+
+start(_StartType, _StartArgs) ->
```

Comment:
If there is no good way not to have this empy behaviour, I would put a brief explanation here.